MANIFESTO.md

```diff
@@ -1 +1,6 @@
 
+##Community
+
+Mankind did not evolve by sitting alone in the dark. For the software industry to really grow in the Caribbean we need collaboration, knowledge and skill sharing. Avoid platforms and tools that do not respect communities that form around them or worse; shun them.
```

Comment:
The connection between local collaboration and types of tools and platforms isn't obvious. Maybe you can provide an example of tools that have a negative impact on communities or explain how these tools have the potential to do this type of damage to the Caribbean software industry.